src/System.Management.Automation/utils/FuzzyMatch.cs

```diff
@@ -23,14 +23,17 @@ public static bool IsFuzzyMatch(string string1, string string2)
 
 
         /// <summary>
-        /// Compute the distance between two strings.
+        /// Compute the case-insensitive distance between two strings.
         /// Based off https://www.csharpstar.com/csharp-string-distance-algorithm/.
         /// </summary>
         /// <param name="string1">The first string to compare.</param>
         /// <param name="string2">The second string to compare.</param>
         /// <returns>The distance value where the lower the value the shorter the distance between the two strings representing a closer match.</returns>
         public static int GetDamerauLevenshteinDistance(string string1, string string2)
         {
+            string1 = string1.ToUpperInvariant();
+            string2 = string2.ToUpperInvariant();
```

Comment:
Just a question: Is it OK to use `Invariant` here? I'm not familiar with this code, but isn't it interactive and user-facing?

Reply:
There is two kind of strings - command names and file names. My thoughts was that it is special identifiers and we could ignore culture. 
Although I do not have strict preferences.